metapackages/auto-instrumentations-node/package.json

```diff
@@ -47,33 +47,33 @@
   },
   "dependencies": {
     "@opentelemetry/instrumentation": "^0.28.0",
-    "@opentelemetry/instrumentation-amqplib": "0.28.0",
-    "@opentelemetry/instrumentation-aws-lambda": "0.30.0",
-    "@opentelemetry/instrumentation-aws-sdk": "0.6.0",
-    "@opentelemetry/instrumentation-bunyan": "0.27.1",
-    "@opentelemetry/instrumentation-cassandra-driver": "0.27.1",
-    "@opentelemetry/instrumentation-connect": "0.27.1",
-    "@opentelemetry/instrumentation-dns": "0.27.1",
-    "@opentelemetry/instrumentation-express": "0.28.0",
-    "@opentelemetry/instrumentation-fastify": "0.26.0",
-    "@opentelemetry/instrumentation-generic-pool": "0.27.2",
-    "@opentelemetry/instrumentation-graphql": "0.27.4",
+    "@opentelemetry/instrumentation-amqplib": "^0.29.0",
```

Comment:
This is probably not what we wanted. I wonder if it's an easy fix or if we should release the current PR and open an issue to look it later

Reply:
I guess we can but then release please will restore it on the next PR.
I am fine with publishing current PR and adding an issue to look into it later

Reply:
> I guess we can but then release please will restore it on the next PR.

Yes, but in that way the package would have pinned ranges in the released version.

Reply:
This seems to be built in into release please currently, see https://github.com/open-telemetry/opentelemetry-js-contrib/pull/947#issuecomment-1126018671